SU2_CFD/include/driver_structure.hpp

```diff
@@ -355,12 +356,12 @@ class CSingleZoneDriver : public CDriver {
 
 
 /*!
- * \class CMultiZoneDriver
+ * \class CFluidDriver
```

Comment:
I am fine with the name CFluidDriver for now. The CDriver class is the highest level of the multi-physics problem definition. It is easy to add new child classes containing different physics/couplings, so it's unlikely that we'll have an easy pattern for naming at this level. As long as it is descriptive of the problem at the highest level considering all components, it is ok.